components/sensor/sgp30.rst

```diff
@@ -72,12 +72,12 @@ Advanced:
   - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
   - All other options from :ref:`Sensor <config-sensor>`.
 
-- **compensation** (*Optional*): The block containing sensors used for compensation.
+- **compensation** (*Optional*): The block containing sensors used for compensation. Both values should be supplied in order to be able to generate the absolute humidity to be reported to the sensor.
 
   - **temperature_source** (*Optional*, :ref:`config-id`): Give an external temperature sensor ID
-    here. This can improve the sensor's internal calculations.
+    here. The data should be in Celcius. This can improve the sensor's internal calculations.
```

Comment:
Cel*s*ius
```suggestion
    here. The data should be in Celsius. This can improve the sensor's internal calculations.
```
Also maybe explain why? To me "should" means that it would still work in other units, so what is the advantage of using Celsius? Or is this actually a "must" to get meaningful values at all?